### PR TITLE
Cleanup .gitignore file

### PR DESCRIPTION
**Pull request by shulard**
I am targetting this branch, because this is a pedantic branch.

Related to: sonata-project/dev-kit#211
## Subject

Remove plateform specific ignore entries from .gitignore file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
+/Resources/doc/_build
+/vendor
+/node_modules
+/.sass-cache
 composer.lock
-vendor
-node_modules
-.sass-cache
+phpunit.xml
 .php_cs.cache
```
Comment (core23) [inline .gitignore]: Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore

File: .gitignore [hunk 2]
```diff
@@ -1,12 +1,8 @@
...
-nbproject
-coverage
-composer.lock
-vendor
-node_modules
-.sass-cache
+/.sass-cache
+/build
+/Resources/doc/_build
+/node_modules
```
Comment (core23) [inline .gitignore]: Why?

Comment (shulard) [inline .gitignore]: Though that these folders are project specific. There is SCSS files and a package.json un the Resources folder...

Comment (core23) [inline .gitignore]: Not sure how and where we update our assets. There are not docs or build script atm :(

I think we should remove these files from the `.gitignore` file for now and create a new PR which adds the missing parts for the assets and the coresponding `.gitignore` entries.

Can you do this @shulard ?

Comment (core23) [inline .gitignore]: Refs https://github.com/sonata-project/dev-kit/pull/213

**Comment by shulard** [comment]
I've updated the gitignore file...

**Pull request merged**